### ISSUE TITLE
Question about modpack inclusion

### ISSUE THREAD

**Issue by pajicadvance** [lonefelidae16/extra-sounds#14]
### Describe your suggestion

Sorry for reaching out to you via the issue tracker, but I couldn't find any other way to contact you.

I already saw that you aren't planning on publishing this port on any mod platform. Would it be fine for me to include this port in a Modrinth modpack that I'm making as an override? Thanks for your time.

**Comment by lonefelidae16** [comment]
Thanks for creating this issue.

Sorry I’m not quite familiar with the Modrinth modpack, and “an override” mechanism. I did a little research and interpreted modpack as automatically downloading and installing Jars published on Modrinth from the reference data via a dedicated app, and override as injecting custom data or overwriting default settings or bundled assets. I’ll respond on that basis, but I apologize if I’m off the mark.

Custom builds, not just those distributed in this repo, can include this port in your modpack as an override. However, there are a few things to note.

- I accept no responsibility for any actions you take. Please do so at your own risk. However, I may make an exception when an issue is raised about mod compatibility.
- The lifetime of this port is until the original repo is back. Therefore, I haven’t edited the contact section in [`fabric.mod.json#10`](https://github.com/lonefelidae16/extra-sounds/blob/a8835a2ac4edf46da6a5ef87044f07ab0748348e/src/main/resources/fabric.mod.json#L10-L14), which is referenced by mods like ModMenu. Please try not to mislead users.

I’m thinking it’s time to edit the contact URL as I didn’t anticipate the long term and the possibility of this being in the hands of many people...

Have a nice modding!

**Comment by pajicadvance** [comment]
Yes, an override in a Modrinth modpack is any data contained in the mod pack that's not published on Modrinth itself, and is instead obtained from an external source and included directly in the pack file.

I will make sure to point out that what I'm using is an unofficial port of the original mod. Thanks for your response!
